runtime/common/src/main/java/org/apache/nemo/runtime/common/metric/TaskMetric.java

```diff
@@ -261,6 +292,12 @@ public final String getId() {
   public final boolean processMetricMessage(final String metricField, final byte[] metricValue) {
     LOG.debug("metric {} has just arrived!", metricField);
     switch (metricField) {
+      case "streamMetric":
```

Comment:
Hard-coded strings doesn't look good. Maybe we need to refactor it in the future.